[Doc]Refine README.md (#7)
streamnative/apache-pulsar-grafana-dashboard: README.md

The Grafana dashboard docker image is available at
[Docker Hub](https://hub.docker.com/r/streamnative/apache-pulsar-grafana-dashboard).

To use this dashboard image, provide the following environment variables: 

- *PULSAR_PROMETHEUS_URL*: The HTTP URL that points to your prometheus service. For example, `http://<prometheus-host>:9090`.
- *PULSAR_CLUSTER*: The pulsar cluster name. The cluster name is aligned with your prometheus configuration.
  See [Prometheus](#prometheus) for more details.

## Prometheus

To display the metrics correctly with this dashboard, configure your Prometheus server to collect metrics from Pulsar correctly.

> 1. Attach your prometheus service to an extra label - `cluster`. The cluster name is aligned with the `PULSAR_CLUSTER` name you have provided to the grafana dashboard.

Reviewer: > Attach your prometheus service to an extra label - `cluster`.

I am not sure if this is correct.  The original sentence is to describe - configure your prometheus service to attach an extra label - `cluster` to the metrics collected by prometheus.

Author: Sorry, I misunderstand the original sentence. Could we refine it as follows? 
"Configure Prometheus service, and make sure your Prometheus service attaches an extra label `cluster` to the metrics collected from Pulsar cluster."

Reviewer: +1

> 1. Attach your prometheus service to an extra label - `cluster`. The cluster name is aligned with the `PULSAR_CLUSTER` name you have provided to the grafana dashboard.

Author: ```suggestion
1. Configure Prometheus service, and make sure your Prometheus service attaches an extra label `cluster` to the metrics collected from Pulsar cluster. The cluster name is aligned with the `PULSAR_CLUSTER` name you have provided to the grafana dashboard.
```

- *PULSAR_CLUSTER*: The pulsar cluster name. The cluster name is aligned with your prometheus configuration.
  See [Prometheus](#prometheus) for more details.

## Prometheus

To display the metrics correctly with this dashboard, configure your Prometheus server to collect metrics from Pulsar correctly.

1. Attach your prometheus service to an extra label - `cluster`. The cluster name is aligned with the `PULSAR_CLUSTER` name you have provided to the grafana dashboard.
   ```yaml
   global:
     ...
     external_labels:
       cluster: <your-cluster-name>
   ```

2. Make sure the job name of each component is the same with the ones in this dashboard.
   - job *proxy*: the machines that run pulsar proxies.
   - job *broker*: the machines that run pulsar brokers.
   - job *bookie*: the machines that run bookies.
   - job *zookeeper*: the machines that run zookeeper.
   - job *node_metrics*: all the machines of the pulsar cluster.

How to configure your prometheus server to collect the metrics of a Pulsar cluster, refer to [example prometheus config](prometheus/cluster.yml.template).

## Usage

Use this Grafana Dashboard on a standalone cluster.

> <!--incomplete info-->

Author: ```suggestion
```

- *PULSAR_CLUSTER*: The pulsar cluster name. The cluster name is aligned with your prometheus configuration.
  See [Prometheus](#prometheus) for more details.

## Prometheus

To display the metrics correctly with this dashboard, configure your Prometheus server to collect metrics from Pulsar correctly.

1. Attach your prometheus service to an extra label - `cluster`. The cluster name is aligned with the `PULSAR_CLUSTER` name you have provided to the grafana dashboard.
   ```yaml
   global:
     ...
     external_labels:
       cluster: <your-cluster-name>
   ```

2. Make sure the job name of each component is the same with the ones in this dashboard.
   - job *proxy*: the machines that run pulsar proxies.
   - job *broker*: the machines that run pulsar brokers.
   - job *bookie*: the machines that run bookies.
   - job *zookeeper*: the machines that run zookeeper.
   - job *node_metrics*: all the machines of the pulsar cluster.

How to configure your prometheus server to collect the metrics of a Pulsar cluster, refer to [example prometheus config](prometheus/cluster.yml.template).

## Usage

> Use this Grafana Dashboard on ... <!--on what? please provide details.-->

Reviewer: > <!--on what? please provide details.-->

Use this Grafana Dashboard on a Standalone Cluster. It is breaking into the section `... a Standalone Cluster`.

Author: ok， i'll refine this soon.

> Use this Grafana Dashboard on ... <!--on what? please provide details.-->

Author: ```suggestion
Use this Grafana Dashboard on a standalone cluster.
```

> Use this Grafana Dashboard on ... <!--on what? please provide details.-->

Author: ```suggestion
Use this Grafana Dashboard on a standalone cluster.
```

> ### ... a Standalone Cluster

Author: ```suggestion
```

> ### ... a Standalone Cluster

Author: ```suggestion
```
